libs/geometry/src/TangentSpaceMesh.cpp

```diff
@@ -52,24 +53,22 @@ struct TangentSpaceMeshOutput {
     size_t triangleCount = 0;
     size_t vertexCount = 0;
 
-    float4* tangents = nullptr;
-    float2* uvs = nullptr;
-    float3* positions = nullptr;
-    uint3* triangles32 = nullptr;
-    ushort3* triangles16 = nullptr;
+    const quatf* tangentSpace = nullptr;
+    const float2* uvs = nullptr;
+    const float3* positions = nullptr;
+    const uint3* triangles32 = nullptr;
+    const ushort3* triangles16 = nullptr;
```

Comment:
not a super big deal bu we prefer `type const *` rather than `const type *`

Reply:
done. Does it also apply to function parameters?

Reply:
yes